metadata/en-US/full_description.txt

```diff
@@ -0,0 +1,22 @@
+Get your own self-hosted VPN!
+
+AmneziaVPN is a multi-protocol, open-source VPN client that offers the option to set up your own server. 
+
+Free open-source software to create a personal VPN on your server. Helps to access blocked content without revealing privacy even to VPN providers. Specify IP address, credentials for your server and Amnezia will configure it to connect via VPN automatically.
+
+We are not a VPN service, you don't have to connect to our servers and pay us anything. You free to use your own self-hosted VPS or you could purchase any VPS from any provider. To connect, use your own or purchase any VPS from any provider.
+
+Features
+
+• Connect using a config file, text code, by scanning a QR code, or setting up your own server
+• Select and set the protocols manually or use the "Wizard" function to configure them automatically based on the level of censorship in your country.
+• Supports the following VPN protocols on the server: WireGuard, AmneziaWG, OpenVPN, OpenVPN over Cloak, ShadowSocks and IKEv2.
```

Comment:
ShadowSocks and IKEv2 are not supported on Android, so it's better to remove them from this list. @pokamest, or is it better to completely copy the description from google play?

Reply:
Agree